src/test/java/service/FileServiceTest.java

```diff
@@ -0,0 +1,60 @@
+package service;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FileServiceTest {
+    private static final String EXISTING_FILE = "src/test/resources/first.csv";
+    private static final String NON_EXISTING_FILE = "src/test/resources/non-existing-file.csv";
```

Comment:
```suggestion
    private static final String NON_EXISTING_FILE = "invalid/path/to/non-existing-file.csv";
```
make it immediately obvious this is a dummy
also, there is no difference between an invalid path and non existing file - the result is that FileReader isn't able to find this file at this location